lte/gateway/c/session_manager/test/test_stored_state.cpp

```diff
@@ -289,20 +312,23 @@ TEST_F(StoredStateTest, test_stored_session) {
   EXPECT_EQ(stored_monitor.credit.buckets[ALLOWED_TOTAL], 54321);
   EXPECT_EQ(stored_monitor.level, MonitoringLevel::PCC_RULE_LEVEL);
 
-  EXPECT_EQ(stored.imsi, "IMSI1");
-  EXPECT_EQ(stored.session_id, "session_id");
+  EXPECT_EQ(deserialized.imsi, "IMSI1");
+  EXPECT_EQ(deserialized.session_id, "session_id");
   EXPECT_EQ(
-      stored.subscriber_quota_state, SubscriberQuotaUpdate_Type_VALID_QUOTA);
-  EXPECT_EQ(stored.fsm_state, SESSION_RELEASED);
+      deserialized.subscriber_quota_state,
```

Comment:
We should be testing the fields of the deserialized struct here, not the original